dnf-plugins-core.spec

```diff
@@ -412,6 +412,8 @@ PYTHONPATH=./plugins nosetests-%{python3_version} -s tests/
 %license COPYING
 %doc AUTHORS README.rst
 %ghost %{_var}/cache/dnf/packages.db
+%config(noreplace) %{_sysconfdir}/dnf/plugins/copr.conf
+%config(noreplace) %{_sysconfdir}/dnf/plugins/copr.d/*.conf
```

Comment:
Are there some `%{_sysconfdir}/dnf/plugins/copr.d/*.conf` files?  IMO you only want
`%dir %{_sysconfdir}/dnf/plugins/copr.d`